docs/conf.py

```diff
@@ -1,4 +1,10 @@
-extensions = ['sphinx.ext.autodoc', 'jaraco.packaging.sphinx', 'rst.linker']
+extensions = [
+    'sphinx.ext.autodoc',
+    'jaraco.packaging.sphinx',
+    'rst.linker',
+    'sphinx.ext.intersphinx',
+    'sphinx.ext.extlinks',
+]
```

Comment:
Please revert this change, the top part of this module is merged-in from the shared skeleton, and changing it will mess things up for the maintainers.

Reply:
There should be a comment in conf.py warning contributors about the skeleton

Reply:
done

Reply:
Yeah, probably. But it'd need to go to https://github.com/jaraco/skeleton first. This is something @jaraco may want to address.